app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -36,25 +36,24 @@ keywords:
   - Statement
   - Comparability
 ---
-
-Overseas teachers enrich the lives of students and add value to schools and the wider community. The [Department for Education (DfE)](https://www.gov.uk/government/organisations/department-for-education) wants to help Ukrainian refugees in any way we can, and we support and encourage schools and training providers to recruit Ukrainian teachers and teacher trainees. 
+The [Department for Education (DfE)](https://www.gov.uk/government/organisations/department-for-education) supports and encourages Ukrainian teachers and teacher trainees seeking jobs and training places. 
 
 We can help if you’re interested in teaching children aged 5 to 18. If you'd like to teach other age groups, try [teaching in a university](https://nationalcareers.service.gov.uk/job-profiles/higher-education-lecturer) or [teaching in further education](https://www.teach-in-further-education.campaign.gov.uk/). 
 
 ## General information
 
 If you're coming to the UK from Ukraine, [check what you need to do before you travel and what to do after you arrive](https://www.gov.uk/guidance/move-to-the-uk-if-youre-from-ukraine).
 
-### Check if you are eligible for an international relocation payment worth £10,000
+### Get an international relocation payment worth £10,000
 
-The international relocation payment (IRP) is a single payment of £10,000 which the UK government is making available to non-UK trainees and teachers of languages and physics. To be eligible, you must be coming to England to work or train in the 2023 to 2024 academic year. 
+To be eligible for the international relocation payment, you must be coming to England to teach or train to teach languages or physics in the 2023 to 2024 academic year. 
 
 [Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment).
```

Comment:
@laurahermionetennant92 Does it need to have (IRP) if it's in a link? I can't remember!

Reply:
I've taken both second mentions down to the acronym :)